features/reisdocumenten.feature

```diff
@@ -89,4 +89,148 @@ Functionaliteit: Raadplegen van reisdocumenten op basis van een reisdocumentnumm
     | reisdocumentnummer           | toelichting                                       |
     | <script>hello world</script> | reisdocumentnummer bevat ongeldige karakters      |
     | abc123AB                     | reisdocumentnummer is minder dan 9 karakters lang |
-    | abcd1234AB                   | reisdocumentnummer is meer dan 9 karakters lang   |
\ No newline at end of file
+    | abcd1234AB                   | reisdocumentnummer is meer dan 9 karakters lang   |
+
+  Rule: een reisdocumentnummer wordt niet opgenomen wanneer Datum einde geldigheid Nederlands reisdocument (35.50) vandaag is of in het verleden ligt
+    - een reisdocument wordt alleen opgenomen wanneer Datum einde geldigheid Nederlands reisdocument leeg is, volledig onbekend is of in de toekommst ligt
+    - wanneer alleen het jaar en de maand van Datum einde geldigheid Nederlands reisdocument bekend zijn, beschouwen we het reisdocument de hele maand als geldig
```

Comment:
Denk niet dat we dat moeten doen.

Reply:
deze hele regel, de hele feature, moet anders worden geschreven. gaat niet meer om wel/niet leveren van een reisdocument, maar om vullen indicatieGeldig

Reply:
Stel dat alleen de maand en het jaar bekend is (juni 2022). Moeten we dan gedurende de hele maand juni het reisdocument als geldig bestempelen?

Reply:
@CathyDingemanse dit speelt dan alleen in die maand en is denk ik nogal theoretisch. Ik kan me nog iets voorstellen van een onbekende datum van vermissing. Maar een onbekende datum einde geldigheid??? Volgens mij wordt een reisdocument uitgegeven door de Nederlandse overheid. Hoe weet burgerzaken wel dat er een reisdocument bestaat maar kan die niet zeker zijn welke einddatum geldigheid daarop staat?

Deze rule over (gedeeltelijk) onbekende einddatum is denk ik puur voor technische volledigheid opgenomen, zodat het algoritme elke situatie die volgens het datamodel zou kunnen voorkomen kan afhandelen zonder te crashen.

Reply:
Ja, dat kan ik bevestigen. Volgens de HUP moet bij de datum einde geldigheid de datum overgenomen worden die op het reisdocument als einddatum geldigheid staat vermeld. Op NL-paspporten (en voor zover ik weet op alle ICAO paspoorten) is dat altijd een volledige datum. In een theoretische situatie (bijvoorbeeld bij een technisch probleem tijdens het verhuizen van een persoonslijst van de ene naar de andere gemeente) zou er een fout op kunnen treden waardoor de waarde van dit veld corrupt is geraakt en dan vervangen moet worden door een (deels) onbekende datum. Maar die voorziening is alleen in het LO BRP opgenomen om ervoor te zorgen dat in zoveel mogelijk gevallen de persoonslijst verhuisd kan worden. In de verstrekkingen hoeft daar geen rekening mee gehouden te worden. Deze situtatie kan sowieso slechts tijdelijk blijven bestaan. Als zo'n fout optreedt moet de gemeente er zsm voor zorgen dat het gegeven hersteld wordt. Dat is in alle gevallen mogelijk want de betrokken persoon kan opnieuw het reisdocument tonen waar deze datum op staat.

Reply:
Ok, voor alle zekerheid zijn features toegevoegd voor een tijdelijke situatie, hoe kort deze dan ook mag duren.